backend/src/routes/institute-router.js

```diff
@@ -25,11 +24,13 @@ const isValidUiTokenWithStaffRoles = auth.isValidUiTokenWithRoles(
 router.get("/*", getInstituteAPI);
 
 async function getInstituteAPI(req, res) {
-  const token = getBackendToken(req);
+  const token = auth.getBackendToken(req);
```

Comment:
Should be getBackendServiceToken() I think. Use the service token to call out to apis outside of GRAD.

Reply:
Removed line and pushed change, token const wasn't used since getCommonServiceData() handles fetching the service client token.